common/models.py

```
@@ -0,0 +1,77 @@
+import uuid
+
+from django.conf import settings
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+user_model = settings.AUTH_USER_MODEL
+
+
+class IsDeletedManager(models.Manager):
+    def get_queryset(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_queryset`

(ANN201)